tsconfig.base.all.json

```diff
@@ -175,7 +176,7 @@
       "@fluentui/theme-designer": ["packages/react-components/theme-designer/src/index.ts"],
       "@fluentui/tokens": ["packages/tokens/src/index.ts"],
       "@fluentui/workspace-plugin": ["tools/workspace-plugin/src/index.ts"],
-      "@fluentui/react-timepicker-compat": ["packages/react-components/react-timepicker-compat/src/index.ts"]
```

Comment:
is there a reason timepicker is being removed?

Reply:
Hmm I'm not sure, I ran the script to create the new package, not sure why its removing the other package

Reply:
It's not removed, it is just being moved automatically to be in alphabetical order.

Reply:
hah...i even looked up, but not far enough